docs/evaluation/code.md

```diff
@@ -201,4 +201,85 @@ all you need to do is replace `openhands` with `swe_agent` in the command above.
 ### livebench-coding
 
 - Benchmark is defined in [`nemo_skills/dataset/livebench-coding/__init__.py`](https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/dataset/livebench-coding/__init__.py)
-- Original benchmark source is [here](https://huggingface.co/datasets/livebench/coding).
\ No newline at end of file
+- Original benchmark source is [here](https://huggingface.co/datasets/livebench/coding).
+
+### OJBench
+
+- Benchmark is defined in [`nemo_skills/dataset/ojbench/__init__.py`](https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/dataset/ojbench/__init__.py)
+- Original benchmark source is [here](https://github.com/He-Ren/OJBench/tree/main).
+
+#### Data preparation
+
+Before running ns eval, you will need to prepare the data with this command:
+
+```
+ns prepare_data --data_dir=<DATA_DIR> --cluster=<CLUSTER_NAME> ojbench
```

Comment:
Please update pipeline/prepare_data.py to require data_dir for this benchmark to avoid accidental mistakes https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/pipeline/prepare_data.py#L30

Reply:
Done.